### ISSUE TITLE
icbhi official rules for dividing the dataset

### ISSUE THREAD

**Issue by ttom138** [da-luggas/vae-icbhi2017#1]
Hi author, I would like to ask icbhi what are the official rules for dividing the dataset training set and test set? Please let me know.

**Comment by da-luggas** [comment]
Hi @ttom138, thanks for the question!

The official dataset split is a 60/40 split on a recording level. The exact split utilized in the challenge can be found [here](https://bhichallenge.med.auth.gr/sites/default/files/ICBHI_final_database/ICBHI_challenge_train_test.txt).

**Comment by ttom138** [comment]
OK，thanks for your answer.So
only cycles in the exact filename can be used for training or test?

**Comment by da-luggas** [comment]
Yes, that's the idea to prevent information leakage between the training and test set.

**Comment by ttom138** [comment]
Thanks for your help! Much appreciated.
